src/main/java/com/archisacadeny/course/CourseRepository.java

```diff
@@ -17,5 +17,16 @@ public static void deleteCourse(long courseId) {
         }
     }
 
+    public static void update(String courseNumber, Course course ){
+        String query = String.format(
+                "UPDATE courses SET name= '%1$s', number = '%2$s', instructor_id = '%3$s' WHERE number= '"+courseNumber+"'",
```

Comment:
probably we will have more fields in course table so how you are going to update those fields?